taxonomy-editor-frontend/src/pages/editentry/ListTranslations.jsx

```diff
@@ -58,13 +65,23 @@ const ListTranslations = ({ nodeObject, setNodeObject }) => {
         setOpen(false);
     }
 
+    // Used for toggling visibility of a translation language
+    const handleToggleTranslationVisibility = (key) => {
+        const newShownTranslationLanguages = shownTranslationLanguages.includes(key) ? shownTranslationLanguages.filter(obj => !(key === obj)) : [...shownTranslationLanguages, key]
+        setShownTranslationLanguages(newShownTranslationLanguages);
+        newShownTranslationLanguages.sort()
```

Comment:
do we need the `.sort()`? if so can we do it before setting the state? please be aware that .sort() modifies the array that was passed to the state